server/routes/projects/create.js

```diff
@@ -16,12 +17,12 @@ module.exports = function(config, req, res) {
   }
 
   if(!user) {
-    res.redirect(307, "/editor" + qs);
+    res.redirect(307, "/" + locale + "/editor" + qs);
     return;
   }
 
   var project = {
-    title: Constants.DEFAULT_PROJECT_NAME,
+    title: req.gettext(defaultProjectNameKey, req.localeInfo.locale),
```

Comment:
in some places you guard against `localeInfo` being falsy before accessing `localeInfo.locale` - should we do this in all cases, or not at all?